charts/scalardb/values.yaml

```diff
@@ -0,0 +1,111 @@
+# Default values for scalardb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# nameOverride -- String to partially override scalardb.fullname template (will maintain the release name)
+nameOverride: ""
+# fullnameOverride -- String to fully override scalardb.fullname template
+fullnameOverride: ""
+
+scalardb:
+  # scalardb.replicaCount -- Default values for number of replicas.
+  replicaCount: 3
+
+  storageConfiguration:
+    # scalardb.storageConfiguration.contactPoints -- The database contanct point such as a hostname of Cassandra or a URL of Cosmos DB account.
+    contactPoints: cassandra
+    # scalardb.storageConfiguration.contactPort -- The database port number.
+    contactPort: 9042
+    # scalardb.storageConfiguration.username -- The username of the database. For Cosmos DB please leave blank. For Dynamo DB please specify key id here.
+    username: cassandra
+    # scalardb.storageConfiguration.password -- The password of the database. For Cosmos DB, Dynamo DB please specify a secret key here.
+    password: cassandra
+    # scalardb.storageConfiguration.storage -- Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set.
+    storage: cassandra
```

Comment:
I think we can add logLevel like `ledger`.
https://github.com/scalar-labs/helm-charts/blob/eeef41e69ebe5d2a117049a76aa2729f33ff33b2/charts/scalardl/values.yaml#L120 

ref: https://github.com/scalar-labs/scalardb/blob/master/server/README.md#run-1
```
-e JAVA_OPTS=-Dlog4j.logLevel=DEBUG
```